code/4.3shiny_all_data.R

```diff
@@ -28,25 +28,31 @@ file_ger_shape_county <- here("data", "raw", "shape_ger", subfolder, "vg2500", "
 
 
 # ***********************************************************************************************
-#### load data ####
+#### load data and prepare data####
 # read all data files
 data_state <- read.csv2(here("data", "processed", paste0("data_state", ".csv")), row.names = NULL, encoding = "UTF-8", stringsAsFactors = FALSE)
 data_state_combined_all_sources <- read.csv2(here("data", "processed", paste0("data_state_combined_all_sources", ".csv")), row.names = NULL, encoding = "UTF-8", stringsAsFactors = FALSE)
 data_state_yearly <- read.csv2(here("data", "processed", paste0("data_state_yearly", ".csv")), row.names = NULL, encoding = "UTF-8", stringsAsFactors = FALSE)
-data_state_yearly_combined_all_sources<- read.csv2(here("data", "processed", paste0("data_state_yearly_combined_all_sources", ".csv")), row.names = NULL, encoding = "UTF-8", stringsAsFactors = FALSE)
+df <- data_state_yearly
+df$EinheitenTyp <- "All"
+data_state_yearly_combined_all_sources <- df %>% group_by(start_year, ags_federal_state) %>%
+  summarize(n = sum(n), mean = mean(mean), sum = sum(sum)) %>%
```

Comment:
I don't think it is correct to take `mean` of all other `mean` values. Mean should be calculated from the original data since number of plants is important when calculating mean.

Reply:
Thanks for the feedback. You are right.
The datsets that were loaded before was not consolidated to "All". So I did this. But I corrected the creation of this in 2.select_data now.
Will push in some minutes.